rollup.config.js

```diff
@@ -18,35 +18,35 @@ function external(id) {
 const cjs = [
   {
     input,
-    output: { file: `cjs/${pkg.name}.js`, format: 'cjs' },
+    output: { file: `cjs/${pkg.name}.js`, format: 'cjs', exports: 'named' },
```

Comment:
Rollup wants you to be explicit about named exports when you're providing both a default and named export. It still complains if you provide both a default and a named export, but we're ignoring that for now.